app/Services/Manager.php

```diff
@@ -189,19 +204,31 @@ public function collectContestInfo($data)
      *
      * @return object $signups
      */
-    public function getUserSignup($id, $campaign = null, $campaignRun = null)
+    public function getUserSignup($id, $campaign = null, $campaignRun = null, $getMultiple = false)
     {
         $signups = $this->northstar->getUserSignups($id, $campaign);
 
+        if ($getMultiple) {
+            $multipleSignups = [];
+        }
```

Comment:
what's the multiple signups here? So this function gets a signup for a specific campaign and run for a specific user id? So is this multiple signups for different campaigns for a single user?